THIRD-PARTY-LICENSES.md

```diff
@@ -0,0 +1,146 @@
+github.com/99designs/gqlgen                          MIT License
```

Comment:
I'm not so convinced this is a good idea -- it seems destined to get out of date without at least CI checking that it's still up to date.

What about a "make show-licenses" target instead which just renders this from the existing go.mod?